mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
+source include/not_gtid_enabled.inc;
+
+connection slave;
+set global userstat=ON;
+--echo Rows_updated on slave - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
```

Comment:
I see, but then it looks like it's better suited include/assert.inc: as the testcase is maintained going forward (merged across major versions etc), relying on .result file review after any changes is less robust, possible to miss something, whereas assert.inc would fail at once.